docs/src/main/sphinx/routines/examples.md

```diff
@@ -369,3 +369,34 @@ BEGIN
   RETURN r;
 END
 ```
+
+## Date String Parsing example
+
+This routine parses a date string (VARCHAR) into TIMESTAMP. Date strings are commonly represented by ISO 8601 (e.g. 2023-12-01). Date strings can also be represented by the YYYYmmdd and YYYYmmddHH format (e.g. 20230101, 2023010123). For example, Hive day and hourly partitions can use these formats (e.g. '/day=20230101', '/hour=2023010123'). 
```

Comment:
Do not use "e.g." .. replace with "for example" or build it into the sentence .. values such as ... .. and no need to put () in